docs/stdlib/datetime.rst

```diff
@@ -233,20 +230,20 @@ EdgeDB stores and outputs timezone-aware values in UTC.
 
         select <cal::local_date>'2018-05-07';
 
-    Note that when casting from strings, the string should be in ISO
-    8601 date format.
+    When casting :eql:type:`cal::local_date` from strings, the string must be
+    in ISO8601 date format.
 
-    See functions :eql:func:`cal::date_get`, :eql:func:`cal::to_local_date`,
-    and :eql:func:`to_str` for more ways of working with
-    :eql:type:`cal::local_date`.
+    For more information regarding interacting with this type, see
+    :eql:func:`datetime_get`, :eql:func:`cal::to_local_date` and
```

Comment:
```suggestion
    :eql:func:`datetime_get`, :eql:func:`cal::to_local_date`, and
```

Reply:
Just noticed that the function's referenced have changed here as well. Is that intentional? `cal::date_get` was changed to `datetime_get`. Since both of these can be called on a `cal::local_date`, maybe they should both be called out here?

What do you think, @vpetrovykh? Should the "For more information" paragraph be an exhaustive list of every function that can be applied to the data type, or is there a rationale for including a limited selection of those?

On the other hand, maybe since this page is exclusively about datetimes and related functions, we could just leave it as an exercise to the reader to reference those related functions by navigating the table at the top of the page.